File: MultiLanguageSandbox/src/test/deps/swift/49.swift
func determineSign(_ n: Int) -> String {
    if n > 0 {
        return "positive"
    } else if n < 0 {
        return "negative"
    } else {
        return "zero"
    }
}


func testDetermineSign() {
    assert(determineSign(3) == "positive", "3 should be positive.")
    assert(determineSign(0) == "zero", "0 should be zero.")
    assert(determineSign(-5) == "negative", "-5 should be negative.")
    assert(determineSign(1000000000) == "positive", "1000000000 should be positive.")
    assert(determineSign(-1000000000) == "negative", "-1000000000 should be negative.")
    assert(determineSign(1) == "positive", "1 should be positive.")
}

// Call the test function to verify the functionality
testDetermineSign()

File: MultiLanguageSandbox/src/test/deps/elixir/49.exs
defmodule SignChecker do
def determine_sign(n) when is_integer(n) do
    cond do
      n > 0 -> "positive"
      n == 0 -> "zero"
      true -> "negative"
    end
  end
end



ExUnit.start()
defmodule SignCheckerTest do
  use ExUnit.Case

  test "determine_sign returns positive for positive integers" do
    assert SignChecker.determine_sign(1) == "positive"
    assert SignChecker.determine_sign(100) == "positive"
  end

  test "determine_sign returns zero for zero" do
    assert SignChecker.determine_sign(0) == "zero"
  end

  test "determine_sign returns negative for negative integers" do
    assert SignChecker.determine_sign(-1) == "negative"
    assert SignChecker.determine_sign(-100) == "negative"
  end
end